src/test/java/teammates/ui/webapi/GetFeedbackSessionLogsActionTest.java

```diff
@@ -0,0 +1,156 @@
+package teammates.ui.webapi;
+
+import java.util.List;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
+import teammates.ui.constants.LogType;
+import teammates.ui.output.FeedbackSessionLogData;
+import teammates.ui.output.FeedbackSessionLogEntryData;
+import teammates.ui.output.FeedbackSessionLogsData;
+
+/**
+ * SUT: {@link GetFeedbackSessionLogsAction}.
+ */
+public class GetFeedbackSessionLogsActionTest extends BaseActionTest<GetFeedbackSessionLogsAction> {
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.SESSION_LOGS;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @Test
+    @Override
+    protected void testExecute() throws Exception {
```

Comment:
Should probably have a test where start time and end time is not a valid number (like the string "abc")

Reply:
👍